doc/build/html/_sources/rtd_pages/archive_git/release-notes.rst.txt

```diff
@@ -0,0 +1,9 @@
+=============
+Release Notes
+=============
+
+(note: this is a temporary file, to be added-to by anybody, and moved to
+release-notes at release time)
```

Comment:
This is going to be a very hard file to remember to update as we add new notable changes, given it is buried six folders deep.